README.md

```diff
@@ -56,6 +56,7 @@ If you choose to do this make sure you update the `data-generator.yaml` file for
 5. Install cert-manager (this creates cert-manager in a namespace called `cert-manager`):
    ```
    kubectl create -f https://github.com/jetstack/cert-manager/releases/download/v1.15.2/cert-manager.yaml
+   kubectl wait deployment --all  --for=condition=Available=True --timeout=300s -n cert-manager
```

Comment:
There are 3 deployments as part of cert-manager installation. In the test we wait for all of them just to be sure, guess it might be better to the same here or wait directly for webhook api (it is suggested by cert-manager docs)

Reply:
I'm passing the `--all` flag which I understand awaits all deployments to be ready.

Reply:
Yeah but it does not mean that cert manager is really ready. We also wait in test suite one more minute to be sure that cert manager can serve certs.

Reply:
> I'm passing the --all flag which I understand awaits all deployments in the namespace to be ready.

Oh sorry, I saw `-n` and thought it is for name.

Reply:
> Yeah but it does not mean that cert manager is really ready.

That'd be a cert manager bug, but let's not worry about that now!

Despite that, I still think the `wait deployment' is a useful addition as it lessens the chance of the `helm install` failing.